app/src/main/java/com/quester/experiment/dagger2experiment/engine/state/PersistentGameObject.java

```diff
@@ -1,10 +1,34 @@
 package com.quester.experiment.dagger2experiment.engine.state;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 /**
  * Created by Josip on 14/01/2015!
  */
 public class PersistentGameObject {
 
-    //TODO: implement
+    private final String stringRepresentation;
+
+    private PersistentGameObject(String stringRepresentation) {
+        this.stringRepresentation = stringRepresentation;
+    }
+
+    public static PersistentGameObject fromString(String stringRepresentation) throws JSONException {
+        new JSONObject(stringRepresentation);
```

Comment:
nije mi jasno što ova linija radi, ništa ne vraća, ako samo testira može li se pretvoriti u json stavi u metodu validate